.github/workflows/pr.yml

```diff
@@ -1528,6 +1529,186 @@ jobs:
           if [ "$numFail" -gt 0 ];then echo "*** $numFail Failed ***";exit 2;fi
           if [ "$numPass" -eq 0 ];then echo "*** 0 Passed! ***";exit 2;fi
           if [ "$numPass" -lt 9 ];then echo "*** Failed to pass at least 9 ! ***";exit 2;fi
+
+  pkg_install_amd64:
```

Comment:
created new github jobs that only run on ok-to-test label
and also change the name.

Reply:
done